atomic_reactor/util.py

```diff
@@ -915,15 +921,11 @@ def get_config_and_id_from_registry(self, image, digest, version='v2'):
 
         :return: dict, versions mapped to their digest
         """
-        response = query_registry(
-            self._session, image, digest=digest, version=version)
+        response = query_registry(self._session, image, digest=digest, version=version)
         manifest_config = response.json()
-        config_digest = manifest_config['config']['digest']
-
-        config_response = query_registry(
-            self._session, image, digest=config_digest, version=version, is_blob=True)
 
-        blob_config = config_response.json()
+        config_digest = manifest_config['config']['digest']
+        blob_config = self._config_by_digest(image, config_digest)
```

Comment:
Oh true, I didn't notice 'v1' was the default. I don't think it matters when querying the blob config, because both v1 and v2 are wrong:

v1 -> `application/vnd.docker.distribution.manifest.v1+json`
v2 -> `application/vnd.docker.distribution.manifest.v2+json`

but the content type of blob config is `application/vnd.docker.container.image.v1+json`

Reply:
But yeah, we probably should eventually drop v1, because OSBS will not work correctly with it anyway